### PR TITLE
new version - filezilla 3.11.0

### PR DESCRIPTION
**Pull request by Philosoft**

### CODE REVIEW

File: Casks/filezilla.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@
     version '3.8.1'
     sha256 '86c725246e2190b04193ce8e7e5ea89d5b9318e9f20f5b6f9cdd45b6f5c2d283'
   else
-    version '3.10.3'
-    sha256 '63541098c881dd698aa648506cf922b857d33a12fa31ad39d6faa851f5d5d49b'
+    version '3.11.0'¬
```
Comment (vitorgalvao) [inline Casks/filezilla.rb]: There’s an extra character, there.

File: Casks/filezilla.rb [hunk 2]
```diff
@@ -4,8 +4,8 @@
...
+    sha256 '552fa74acc7fb39a909173ecb907abaa9f970a807b9fe5be40607eaa6d3021e1'¬
```
Comment (vitorgalvao) [inline Casks/filezilla.rb]: Also here.

**Comment by winkelsdorf** [comment]
Can be closed, 3.11.0.1 already merged.

**Comment by Philosoft** [comment]
such a looser - copy'n'paste with visual line terminators, sorry for trouble :( been out of town and internet so didn't react fast enough.

**Pull request closed without merging**